build.jam

```diff
@@ -0,0 +1,27 @@
+# Copyright René Ferdinand Rivera Morell 2023
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file LICENSE_1_0.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+
+require-b2 5.2 ;
+
+project /boost/atomic
+    : common-requirements
+        <include>include
+        <library>/boost/align//boost_align
+        <library>/boost/assert//boost_assert
+        <library>/boost/config//boost_config
+        <library>/boost/predef//boost_predef
+        <library>/boost/preprocessor//boost_preprocessor
+        <library>/boost/type_traits//boost_type_traits
+        <library>/boost/winapi//boost_winapi
```

Comment:
`winapi` is only needed on Windows.

Also, shouldn't the public and private dependencies be specified separately? And for the library itself rather than the project?

Reply:
That report doesn't separate public and private dependencies, it lists everything that is used in headers and source files together.

You can use `CMakeLists.txt` to deduce the actual public and private dependencies. Boost.Built should work equivalently.